app/templates/components/payments/funds-recipient/verification-document.hbs

```diff
@@ -15,7 +15,10 @@
         uploadDone=(action onUploadDone)
         uploadError=(action onUploadError)
         validationError=(action onValidationError)}}
-    </div>
+    </p>
+    {{#if (eq status 'errored')}}
+      <p class="error">Stripe was unable to verify your identity with the document you provided. Please try again.</p>
+    {{/if}}
     {{#if error}}
       <div class="error">{{error}}</div>
```

Comment:
We got a `did.error` and a `p.error` here. Can they be shown at the same time and are we ok with how the margins are in that case?